.scripts/deploynaut-php-resque

```diff
@@ -79,6 +80,27 @@ do_stop()
 	return "$RETVAL"
 }
 
+#
+# Does some checks before stopping to ensure no tasks are running.
+# This will force a loop until no tasks are running.
+#
+pre_stop_check()
```

Comment:
This still allows the possibility for someone to start a job before you manage to kill off the process.

It seems it would be much safer to use signals - [check out the Resque's QUIT signal handler](https://github.com/chrisboulton/php-resque/blob/master/lib/Resque/Worker.php#L343) . The `start-stop-daemon` in do_stop above is already sending signals, it's just it's using the wrong ones :-)